include/picongpu/particles/Manipulate.hpp

```diff
@@ -20,21 +20,48 @@
 #pragma once
 
 #include "picongpu/simulation_defines.hpp"
-#include "picongpu/particles/filter/filter.def"
 #include "picongpu/particles/manipulators/manipulators.def"
+#include "picongpu/particles/filter/filter.def"
 
-#include <pmacc/Environment.hpp>
+#include <pmacc/particles/algorithm/CallForEach.hpp>
 #include <pmacc/meta/conversion/ToSeq.hpp>
 #include <pmacc/meta/ForEach.hpp>
 #include <pmacc/particles/meta/FindByNameOrType.hpp>
-
-#include <boost/mpl/apply.hpp>
+#include <boost/mpl/placeholders.hpp>
+#include <boost/mpl/placeholders.hpp>
```

Comment:
Ah, my earlier comment might have been to outdated code. Now the `placeholders.hpp` is included twice.